sdks/python/apache_beam/pipeline_test.py

```diff
@@ -520,11 +520,11 @@ def test_dir(self):
     options = Breakfast()
     self.assertEquals(
         set(['from_dictionary', 'get_all_options', 'slices', 'style',
-             'view_as', 'display_data']),
+             'view_as', 'display_data', 'next']),
```

Comment:
This is fine, but was there a particular reason it was added?

Reply:
This is because of the import `from builtins import object` in apache_beam/transforms/display.py. 

This import adds an alias: 
`next = __next__` to make Python2 and Python3 object classes compatible

PipelineOptions (the tested class in this test) inherits from HasDisplayData class defined in the display.py module.